build/deploy

```diff
@@ -3,6 +3,7 @@
 git checkout master && \
   git pull origin master && \
   git checkout gh-pages && \
+  git pull origin gh-pages && \
   git merge master && \
```

Comment:
The merge command should have a `--no-edit` on it otherwise it spawns the user's editor when it's a non-ff merge.

Reply:
I think that's another point in favour of doing a reset instead of a pull